network/src/commonMain/kotlin/com/wire/kalium/network/api/user/LegalHoldStatusResponse.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.kalium.network.api.user
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+enum class LegalHoldStatusResponse {
```

Comment:
i have seen this one before 👀
it's hidden tho
https://github.com/wireapp/kalium/blob/bedee9b7f21c0c6665f4a78117734c0e38cfcce0/network/src/commonMain/kotlin/com/wire/kalium/network/api/teams/TeamsApi.kt#L47

Reply:
It wasn't supposed to be in this PR :P

I had it extracted in another branch and mixed up some diffs.
I'll remove it. It's not needed here.